dacite/types.py

```diff
@@ -37,6 +37,14 @@ def transform_value(
     return value
 
 
+def get_data_class_hints(data_class, globalns) -> dict:
```

Comment:
We need tests for this function. If it's possible we should also provide type hints for `data_class` and `globalns` and more specific type hint for return value.